css/properties/height.json

```diff
@@ -51,51 +51,48 @@
             "deprecated": false
           }
         },
-        "stretch": {
+        "max-content": {
```

Comment:
I'm not up to speed on these features. Has this file been alphabetized or is this a name change?

Reply:
I've added these features to the file. I don't know why it's showing up like that.

Reply:
Yeah I think this is just git being too clever about the diff—flipping between this file on `master` and this revision makes it a bit easier to follow. Resolving this.